src/DssSpell.sol

```diff
@@ -59,187 +51,241 @@ contract DssSpellAction is DssAction {
+            dstBuf: 400 * MILLION * WAD,
 
-        // ---------- Aligned Delegate MKR Compensation ----------
-        // Forum: https://forum.makerdao.com/t/august-2024-aligned-delegate-compensation/25165
-        // Mip: https://mips.makerdao.com/mips/details/MIP101#2-6-3-aligned-delegate-budget-and-participation-requirements
+            // LITE-PSM-USDC-A DC-IAM line: Increase by 2,5 billion DAI, from 7,5 billion DAI to 10 billion DAI.
+            dstMaxLine: 10 * BILLION * RAD,
 
-        // BLUE - 13.75 MKR - 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf
-        MKR.transfer(BLUE, 13.75 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+            // LITE-PSM-USDC-A DC-IAM gap: Increase by 200 million DAI, from 200 million DAI to 400 million DAI.
+            dstGap: 400 * MILLION * RAD,
 
-        // Cloaky - 12.00 MKR - 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818
-        MKR.transfer(CLOAKY, 12.00 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+            // LITE-PSM-USDC-A DC-IAM ttl: 12h (Unchanged)
+            dstTtl: 12 hours,
 
-        // JuliaChang - 1.25 MKR - 0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7
-        MKR.transfer(JULIACHANG, 1.25 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+            // Note: chainlog key of PSM-USDC-A
+            srcPsmKey:  "MCD_PSM_USDC_A"
 
-        // Byteron - 1.25 MKR - 0xc2982e72D060cab2387Dba96b846acb8c96EfF66
-        MKR.transfer(BYTERON, 1.25 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        });
 
-        // Rocky - 1.21 MKR - 0xC31637BDA32a0811E39456A59022D2C386cb2C85
-        MKR.transfer(ROCKY, 1.21 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // Note: LitePSM migration was extracted into a library,
+        //       and implemented as part of the LitePSM module.
+        DssLitePsmMigrationPhase3.migrate(dss, migrationCfg);
 
-        // ---------- Spark Spell ----------
-        // Forum: https://forum.makerdao.com/t/sep-12-2024-proposed-changes-to-spark-for-upcoming-spell/25076
-        // Poll: https://vote.makerdao.com/polling/QmPFkXna
+        // ---------- Activate Aave Lido Market USDS D3M  ----------
+        // Forum: https://forum.sky.money/t/risk-assessment-and-parameter-recommendations-spark-ddm-to-aave-lido-market/25175
 
-        // Execute Spark Proxy Spell at 0xc80621140bEe6A105C180Ae7cb0a084c2409C738
-        ProxyLike(SPARK_PROXY).exec(SPARK_SPELL, abi.encodeWithSignature("execute()"));
+        // Set D3M DC-IAM with the following parameters:
+        // line: 100 million USDS
+        // gap: 50 million USDS
+        // ttl: 24 hours
+        // tau: 7 days
+        // D3M Addresses:
+        // oracle: 0x9D9CD271C9f203375b96673056BB20BcC0526E80
+        // plan: 0x4Cb3f51b97D64C122fC52B3CA828516B5FD66EF7
+        // pool: 0x077B5B4b14ebbEF0DAeE21cfAc4CE14523576E07
```

Comment:
Comments doesn't match

Reply:
fixed in https://github.com/makerdao/spells-mainnet/pull/432/commits/b9f1c6a61f16a2006952aa42dd6fa8b2fb38a959